components/PersonWidget/PersonWidget.tsx

```diff
@@ -30,14 +32,11 @@ const PersonWidget = ({ person }: Props): React.ReactElement => {
         {dateOfBirth && (
           <p className={`lbh-body-s ${s.paragraph}`}>Born {dateOfBirth}</p>
         )}
-
-        {displayAddress && (
-          <p className={`lbh-body-s ${s.paragraph}`}>
-            {displayAddress.addressLines}
-            <br />
-            {displayAddress.postCode}
-          </p>
-        )}
+        <p className={`lbh-body-s ${s.paragraph}`}>
+          {displayAddress || displayAddresses?.addressLines}
```

Comment:
is it ever possible to have no address on a person at all?

Reply:
maybe but i thought all the question marks in the consts would deal with that? Do they not?

Reply:
you'd still end up with an empty div if there is no address to show - if it's not possible for a person to have no address (if it's always a required field) then that's fine